app/assets/javascripts/components/wizard/option.jsx

```diff
@@ -16,13 +16,18 @@ const Option = createReactClass({
     selectWizardOption: PropTypes.func.isRequired
   },
 
+  getInitialState() {
+    this.descriptionRef = React.createRef(null);
```

Comment:
Is `getInitialState` the best place for this, or should it be in a different lifecycle function?

Reply:
This should be done in the constructor but because we're not using ES6 classes that doesn't exist. In [the docs](https://reactjs.org/docs/react-without-es6.html#setting-the-initial-state), they used `getInitialState` in place of the constructor so that's what I went with.

Reply:
Cool.